catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -401,10 +401,7 @@ public static void setAutoCrashReportingEnabled(Context context, boolean isEnabl
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, isEnabled);
 		editor.commit();
-
-		if (isEnabled) {
-			CrashReporter.initialize(context);
-		}
+		CrashReporter.IS_CRASH_REPORTER_INITIALIZED = CrashReporter.initialize(context);
```

Comment:
Instead of assigning the variable here, update the variable in the `CrashReporter.initialize` method itself.